components/announcement_bar/announcement_bar.jsx

```diff
@@ -38,12 +40,8 @@ export default class AnnouncementBar extends React.PureComponent {
         }
     }
 
-    setBodyClass = (fixed) => {
-        if (fixed) {
-            document.body.classList.add('announcement-bar--fixed');
-        } else {
-            document.body.classList.remove('announcement-bar--fixed');
-        }
+    setBodyClass = () => {
+        document.body.classList.add('announcement-bar--fixed');
```

Comment:
We can just move this to `componentDidMount()` since there is no logic here anymore.